### PR TITLE
QUESTLV100 cleanup, Early game lines touch up.

### PR DESCRIPTION
**Pull request by ttgmichael**
Finally got the time to do some editing. :)

I checked early game lines and tried to make it sound more natural line by line.
This PR also cleans up a lot of duplicate lines that I found in QUESTLV100. (My god, there were a lot of them xD)

Hope I can help a bit more in the future. Haha

### CODE REVIEW

**Comment by Sunaries** [comment]
What's with the many deletions? Can you post your changes based on the current files. It is a bit confusing for me now.

**Comment by ttgmichael** [comment]
@Sunaries 
The deletions are all in the following commits: 

https://github.com/Treeofsavior/EnglishTranslation/pull/1548/commits/dbe69b1592cb66a7a5ac1767541924785e450372

https://github.com/Treeofsavior/EnglishTranslation/pull/1548/commits/8ff45cee3a7a09d8892c7b0dd606326ff1a051ff

Basically, the file had 2 copies of:
(From first link)
Line 12325
Line 12326
Line 12327
Lines 12328-12959

(From second link)
Lines 10669-10675
Lines 12181-12185
Lines 12190-12196
Line 12222
Line 12223
Lines 12961-12975
Lines 12977-13008
Line 13594
Line 14821
Line 14822
Line 14826
Line 14827
Line 14829
Line 14830

Hope this helps!

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -11345,6 +11337,7 @@ QUEST_LV_0100_20151224_011336	Well, it's no big issue.{nl}Everything has a reaso
 QUEST_LV_0100_20151224_011337	The Royal Mausoleum must contain a secret bigger than we could possibly imagine.{nl}The next epitaph is at Serno Highland. Let's go.
 QUEST_LV_0100_20151224_011338	As Laulas's injury are severe, I'm here in his stead.{nl}The seal is released, what exactly happened here?{nl}
 QUEST_LV_0100_20151224_011339	Whatever the case may be, the altars here are one of the{nl}Royal Mausoleum's various defenses built to ward off intruders.{nl}They should, however, the seal at Rukas Plateau is the answer.{nl}The altar won't function if it gets destroyed, but if we can activate it before that.{nl}
+QUEST_LV_0100_20151224_011340
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:11340]: The `They should, however, the seal at Rukas Plateau is the answer.{nl}The altar won't function if it gets destroyed, but if we can activate it before that` part from 11339 should be in 11340

**Comment by Sunaries** [comment]
I don't see the entered text for `QUEST_LV_0100_20151224_011340`?
Seems to be blank on your repository.

**Comment by ttgmichael** [comment]
@Sunaries 
Line 11340 was missing the whole time! I added what I think is the translation of that line.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**